tidb-cloud/migration-job.md

```diff
@@ -0,0 +1,161 @@
+
+## Step 1: Go to the **Data Migration** page
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com/console/clusters). Navigate to the **Clusters** page for your project.
+
+2. On the **Clusters** page, click the name of your cluster, and click the **Data Migration** tab. The **Data Migration** page is displayed.
+
+3. On the **Data Migration** page, click **Create Migration Job**. The **Create Migration Job** page is displayed.
+
+## Step 2: Configure the source and target connection
+
+On the **Create Migration Job** page, configure the source and target connection.
+
+1. Create a job name. It must start with a letter, and can consist of letters (A-Z, a-z), numbers (0-9), underscores (_) and hyphens (-) with less than 60 characters.
+
+2. Fill in the source connection profile.
+
+   - **Data source**: the data source type. Currently, it supports MySQL, AWS Aurora MySQL, and AWS RDS MySQL.
+   - **Region**: the region of the data source. If the source database is self-built, the parameter is optional.
+   - **Connectivity method**: the connectivity method of the data source. Currently, it supports Public IP, VPC Peering and Private Link.
+   - **Hostname or IP address** (for Public IP): the hostname or IP address of the data source.
+   - **Service Name** (for Private Link and VPC Peering): the endpoint service name.
+   - **Port**: the port of the data source.
+   - **Username**: the username of the data source.
+   - **Password**: the password of the username.
+   - **SSL/TLS**: enable or disable the SSL/TLS configuration of the data source. If you enable SSL/TLS, you need to upload the certificate of the data source, including the CA certificate, client certificate, and client key.
+
+3. Fill in the target connection profile.
+
+   - **Username**: enter the username of the target cluster.
+   - **Password**: enter the password of username.
+
+4. Click **Validate Connection and Next** to validate the information you have entered.
+
+5. A message is displayed.
+
+    - If you use Public IP or  VPC Peering, you are prompted to add Data Migration service's IP addresses to the IP Access List of your source database and firewall (if any) to allow the Data Migration service to access your source database.
+    - If you use Private Link, you are prompted to accept the endpoint request in your account.
+
+## Step 3: Choose the objects to be migrated
+
+1. You can choose to perform full data migration, incremental data migration, or both. If you want to migrate data to TiDB Cloud and switch to TiDB Cloud from now on, it is recommended to select both full data migration and incremental data migration to ensure data consistency between the source and target databases. If you only select the full data migration checkbox, the migration job only migrates the existing data of the source database.
```

Comment:
Since the scenarios of both modes are explained in Step 2, it's a bit redundant here as Step 1. In addition, I did not see the "full data migration" box in the interface.
![image](https://user-images.githubusercontent.com/41534398/202377379-19ba8f68-4fe1-4b13-a783-7104ac0e8cc8.png)

Reply:
updated screenshots